api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -85,11 +85,27 @@ def search_items_with_identifier(
             query_kwargs.update(ExclusiveStartKey=start_key)
         return self.query_items(**query_kwargs)
 
-    def is_migration_done(self, project_id: int) -> bool:
+    def get_migration_status(self, project_id: int) -> str:
         project_metadata = DynamoProjectMetadata.get_or_new(project_id)
-        return project_metadata.is_identity_migration_done
+        return project_metadata.identity_migration_status
+
+    def is_migration_done(self, project_id: int) -> bool:
+        migration_status = self.get_migration_status(project_id)
+        return (
+            migration_status == ProjectIdentityMigrationStatus.MIGRATION_COMPLETED.name
```

Comment:
So, this is my issue with python enums, I don't like that you have to add `name` to the end before doing anything useful with them. 

I wonder if we should in fact just refactor the DynamoProjectMetadata class so that, when we load it (maybe via a marshmallow schema), we do something like: 

```python
def load_identity_migration_status(self, value) -> ProjectIdentityMigrationStatus:
    return ProjectIdentityMigrationStatus(value)
```

That way, we can just do the comparison against the enum itself. 

Alternatively, we could just use constants... I'm not the biggest fan of enums in python.